### PR TITLE
Pa01

### PR DESCRIPTION
**Pull request by jakeriksen**

### CODE REVIEW

**Comment by jakeriksen** [comment]
Removed the copied function, and importing it instead.

File: src/pa01/snakes_and_ladders.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+			player_pos[game] += throw_dice()  # adding dice-value to position
+
+			# Checking if player position is in snake or ladder position
+			# If so, it replaces the position accordingly.
+			if player_pos[game] in snake_ladder:
+				player_pos[game] = snake_ladder[player_pos[game]]
+
+			player_moves[game] += 1  # moves/turns of the player
+
+	winner = min(player_moves)  # player with lowest moves wins
```
Comment (huynhngoc) [inline src/pa01/snakes_and_ladders.py]:
```
num_moves = 0
while max(player_pos) < 90:
    num_moves += 1
    for i, player in enumerate(player_pos):
        # move the players


return num_moves
```

File: src/pa01/snakes_and_ladders.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+	---------
+	num_players : int
+	    Number of players in the game
+
+	Returns
+	-------
+	num_moves : int
+	    Number of moves the winning player needed to reach the goal
+	"""
+	snake_ladder = {1 : 40, 8: 10, 36: 52, 43: 62, 49: 79, 65: 82, 68: 85,
```
Comment (huynhngoc) [inline src/pa01/snakes_and_ladders.py]: You may consider making this variable a constant

File: src/pa01/snakes_and_ladders.py (hunk 1 above)
Comment (jakeriksen) [reply src/pa01/snakes_and_ladders.py]: I see, the code you wrote under is more easy to understand.

**Pull request merged**